doc/Overview.md

```diff
@@ -19,37 +19,153 @@
+
+\section OverviewInstall How to install Pinocchio?
+
+Pinocchio is best installed from APT packaging on Ubuntu 14.04 and 16.04, from our repository.
+Every released is validated in the main Linux distribution and OSX, for which installation from source should be straightforward.
 
 The full installation procedure can be found on the Github Pages of the project:
 http://stack-of-tasks.github.io/pinocchio/download.html.
 
-## Simplest example with compilation command
+\section OverviewSimple Simplest example with compilation command
+
+We start by a simple program to compute the robot inverse dynamics. It is given in both C++ and Python version.
+
+<table class="manual">
+<tr><th>C++ version</th><th>Python version</th></tr>
+<tr><td>
+\include overview-simple.cpp
+</td>
+<td>
+\include overview-simple.py
+</td></tr></table>
+
+\subsection OverviewSimpleCompile Compiling and running your program
+
+You would compile the C++ version by including Pinocchio and Eigen header directories.
+
+\code g++ -I /usr/include/eigen -I /path/to/pinocchio/include sample-rnea.cpp -L /path/to/pinocchio/lib -lpinocchio -o sample-rnea \endcode
+
+You might then run it using
+
+\code ./sample-rnea \endcode
+
+In Python, just run it:
+
+\code python sample-rnea.py \endcode
+
+\subsection OverviewSimpleExplain Explaination of the program
+
+This program load a robot model, create a data structure for algorithm buffering, and run the Recursive-Newton-Euler Algorithm (rnea) to compute the robot inverse dynamics.
+
+We first include the proper files. In C++, there is not yet an rationalization of the include files. Here we have to include the sample model (where the parameters of the model we are using in the test are defined), and the header files defining the robot neutral position and the RNEA function. In Python, the library is easier to include by just importing pinocchio.
+
+The first paragraph defines the model. In C++, we first define the object, then allocate it to be our sample model (a human-like kinematic tree with random parameters, very useful for small tests). In Python, the model is created while calling the sample humanoid model.
+The model class contains the constant parameters of the robot (masses, segment length, body names, etc), i.e. the parameters that are not to be modified by the algorithms. As most algorithms need extra room for storage of internal values, we also have to allocate a Data structure, dedicated to the model. Pinocchio relies on a strict separation between constant parameters, in Model, and calculus buffer, in Data.
+
+Inverse dynamics computes the needed torque to track a trajectory defined by position, velocity and acceleration in the configuration space.
```

Comment:
I would add here to be explicit: 'a trajectory defined by the joint position, velocity and acceleration' and remove in the configuration space.